src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -192,15 +186,18 @@ export default class CodeSnippet extends Component {
       />
     );
 
+    // TODO: Get the popup announced like an alert
+
     if (type === 'inline') {
       return (
         <Copy
           {...other}
           onClick={onClick}
+          aria-label={ariaLabel}
+          aria-describedby="description"
           className={codeSnippetClasses}
-          aria-label={copyLabel}
           feedback={feedback}>
-          <code>{children}</code>
+          <code id="description">{children}</code>
```

Comment:
We need to ensure ID uniqueness - We can generate a unique ID by: https://github.com/IBM/carbon-components-react/blob/v7.0.0/src/tools/uniqueId.js